pyproject.toml

```diff
@@ -45,6 +45,12 @@
     name = "Contributor-facing changes"
     showcontent = true
 
+[build-system]
+requires = [
+  "setuptools>=45",
+  "wheel",
```

Comment:
This is not necessary, I even had to fix setuptools' upstream doc recently to clarify this.
```suggestion
```

Reply:
Without, these warnings are shown:

```
WARNING: Missing build requirements in pyproject.toml for file:///home/bthornto/github/ansible-navigator.
 WARNING: The project does not specify a build backend, and pip cannot fall back to setuptools without 'wheel'.
```

Reply:
Adding this `build-backend = "setuptools.build_meta"` eliminates the WARNINGS

Reply:
That is correct. It is strongly recommended to have `build-backend` set explicitly, otherwise, the fallback is `setuptools.build_meta.__legacy__` and not `setuptools.build_meta` which may behave slightly differently.